### DISCUSSION TITLE
Pull Shark badge<title>

### DISCUSSION THREAD

**Discussion by MidhiaQueiroz** [community/community#67759] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How do I get the new Pull Shark GitHub

**Comment by mehulrajdev** [comment, marked as answer]
To get the achievement, you need to open 2 pull requests that have been merged.
From the 16th PR, you win the bronze medal, after the 128th you win the silver medal and finally, in your 1024th you win the gold medal.

https://github.com/Schweinepriester/github-profile-achievements
